common_interfaces/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!--
```

Comment:
Remove all the message packages.

Reply:
Unfortunately there is an issue with the IDL generation that requires fixing the packages containing message definition and I don't know how to fix this yet (see [https://answers.ros.org/question/315189/fail-at-ros2-idl-generation/](https://answers.ros.org/question/315189/fail-at-ros2-idl-generation/)).

Moreover, when this issue is fixed, message packages might still be required, at least for the example to compile since the ROS2 IDL generation for third party generators will only trigger for messages defined in the current workspace.

Reply:
The current version of `ros2_rust` already supports building messages without importing them or making any changes, as long as they are part of the workspace. Sure, it'd be great if in the future out-of-workspace messages work too, but adding the messages to the `ros2_rust` repo is just unmaintainable.